docs/AppFramework.md

```diff
@@ -1,30 +1,36 @@
 # App Framework Resource Guide
 
-The Splunk Operator provides support for Splunk App and Add-on deployment using the App Framework (Beta Version). The App Framework specification adds support for S3-compatible app repository, authentication, and supports specific Splunk Enterprise cluster and standalone [custom resources](https://splunk.github.io/splunk-operator/CustomResources.html) (CR).
+The Splunk Operator provides support for Splunk app and add-on deployment using the App Framework. The App Framework specification supports configuration management using the Splunk Enterprise cluster and standalone [custom resources](https://splunk.github.io/splunk-operator/CustomResources.html) (CR). 
 
 ### Prerequisites
 
 Utilizing the App Framework requires:
 
-* An Amazon S3 or S3-API-compliant remote object storage location. App framework requires read-only access to the path containing the apps.
-* The remote object storage credentials via a secret, or an IAM role.
-* Splunk Apps and Add-ons in a .tgz or .spl archive format.
-* Connections to the remote object storage endpoint need to be secure using a minimum version of TLS 1.2.
+* An Amazon S3 or S3-API-compliant remote object storage location. The App framework requires read-only access to the path used to host the apps.
+* The remote object storage credentials provided as a secret, or in an IAM role.
```

Comment:
Nit: "provided as a secret" --> "provided as a K8 secret object"
(I know its not your change :-))